src/status_im/chat/handlers/commands.cljs

```diff
@@ -1,13 +1,15 @@
 (ns status-im.chat.handlers.commands
-  (:require [re-frame.core :refer [enrich after dispatch]]
+  (:require [cljs.reader :as reader]
```

Comment:
Yes, I need to parse string representation of command markup from realm in that namespace.